run_iwyu_tests.py

```diff
@@ -183,6 +185,8 @@ def setUp(self):
       'range_for.cc': ['.'],
       're_fwd_decl.cc': ['.'],
       'redecls.cc': ['.'],
+      'relative_include_of_export_3.cc':
+          ['tests/cxx/relative_include_of_export_3_subfolder'],
```

Comment:
Hmm. I can't decide whether it's better to have a dedicated sub-directory for each test, or share the existing `subfolder` (soon to be renamed `subdir` in PR #695). In the interest of shorter names, I think I prefer sharing.

Reply:
Done.